.cloud_build/flutter/README.md

```diff
@@ -4,19 +4,18 @@ To create and upload new Docker images to Google Container Registry, run:
 
 ```bash
 gcloud builds submit . \
---config=cloudbuild.yaml \
+--config=.cloud_build/flutter/cloudbuild.yaml \
 --project=$GCP_PROJECT
 ```
 
 Where `$GCP_PROJECT` is the Google Cloud project name.
 
 This creates and uploads these images:
 
-- gcr.io/$PROJECT_ID/flutter:master
+- gcr.io/$PROJECT_ID/flutter:main
 - gcr.io/$PROJECT_ID/flutter:dev
-- gcr.io/$PROJECT_ID/flutter:beta
 - gcr.io/$PROJECT_ID/flutter:stable
-- gcr.io/$PROJECT_ID/flutter
+- gcr.io/$PROJECT_ID/flutter:latest
```

Comment:
Should this be `beta`? I assume we just want to list `main`, `beta`, and `stable` here.

Reply:
Yep I'll fix that.